jupyterlab/labbox_ephys_widgets_jp/src/registerExtensions.tsx.j2

```diff
@@ -0,0 +1,18 @@
+
+// !begin-code-generation!
+import { ExtensionContext } from './extensions/extensionInterface';
+
+{% for extension in extensions if ('jupyter' in extension.tags) -%}
```

Comment:
Here's the conditional in the .js checking for 'jupyter' among the extension.tags